### PR TITLE
HW1 Selenium completed

### PR DESCRIPTION
**Pull request by foria1405**

### CODE REVIEW

File: MulDivTest.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="SuiteMulDiv">
+    <test name="TestMulDiv">
+        <classes>
+            <class name="hw1.MulTest" />
+            <class name="hw1.DivTest"/>
+        </classes>
+    </test>
+</suite>
```
[the remark this answers is not in the document]
Comment (foria1405) [reply MulDivTest.xml]: Moved testng suites to one particular directory test/resources/suites.

File: pom.xml [hunk 2]
```diff
@@ -9,4 +9,112 @@
...
+                            <suiteXmlFiles>
+                                <suiteXmlFile>SumSubTest.xml</suiteXmlFile>
+                                <suiteXmlFile>MulDivTest.xml</suiteXmlFile>
+                                <suiteXmlFile>SinCosTest.xml</suiteXmlFile>
+                            </suiteXmlFiles>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
```
[the remark this answers is not in the document]
Comment (foria1405) [reply pom.xml:91]: Moved plugin configuration out of profile and in profiles changed only suiteXmlFile using properties.

File: src/test/java/hw1/DivTest.java [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    }
+
+    //This test has failed.
+    // Method public double div() does not contain a check for an attempt to divide by zero
+    @Test(testName = "Zero Division Double",
+            dataProvider = "zeroDivisionDataProviderDouble",
+            expectedExceptions = {NumberFormatException.class})
+    public void zeroDivisionTestDouble(double a, double b) {
+        calculator.div(a, b);
+    }
```
[the remark this answers is not in the document]
Comment (foria1405) [reply src/test/java/hw1/DivTest.java]: Added comparison of division result with Double.POSITIVE_INFINITY constant in the case of dividing a positive number by 0.0.  Moreover, I added a comparison of the division result with Double.NEGATIVE_INFINITY constant in the case of dividing a negative number by 0.0 and added a comparison of the division result with Double.NaN constant in the other case of dividing by 0.0.

**Review by akulebyakin — APPROVED** [review]
Approved. Good job!

**Pull request merged**